CMakeLists.txt

```diff
@@ -63,10 +64,21 @@ if(BUILD_DEPENDENCIES)
 
   message(STATUS "Begin building dependencies.")
 
+  count_true(ENABLED_SSL_OPTION_COUNT
+    USE_OPENSSL
+    USE_MBEDTLS)
+
+  if(ENABLED_SSL_OPTION_COUNT GREATER "1")
```

Comment:
If my input is just cmake -DUSE_MBEDTLS=TRUE .., would it fail? If it fails, is the error message clear enough to let me know that I need to disable openssl by -DUSE_OPENSSL=FALSE?